common/djangoapps/student/models.py

```diff
@@ -1506,10 +1506,13 @@ def change_mode(self, mode):
         """Changes this `CourseEnrollment` record's mode to `mode`.  Saves immediately."""
         self.update_enrollment(mode=mode)
 
-    def refundable(self):
+    def refundable(self, user_already_has_certs_for=None):
         """
         For paid/verified certificates, students may receive a refund if they have
         a verified certificate and the deadline for refunds has not yet passed.
+
+        `user_already_has_certs_for` is an optional param that is a set of
```

Comment:
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings says to use the "Arguments:" format, so I'll make that change. I forgot we had settled on that.